.github/workflows/hybrid_branch_deployments.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, closed]
 env:
-  DAGSTER_CLOUD_URL: ${{ secrets.DAGSTER_CLOUD_HYBRID_URL }}
+  DAGSTER_CLOUD_URL: "https://action-demo-hybrid.dogfood.dagster.cloud"
```

Comment:
This should not be here right?

Reply:
I added it because there was no way to know where to go check for the test results otherwise

Reply:
(It's what it was already set to before - I just changed it so that it's visible instead of buried in a github secret that only ben knew the value of)